src/java/org/apache/cassandra/locator/SimpleSeedProvider.java

```diff
@@ -32,6 +33,7 @@
 public class SimpleSeedProvider implements SeedProvider
 {
     private static final Logger logger = LoggerFactory.getLogger(SimpleSeedProvider.class);
+    private static final int seedCountWarnThreshold = Integer.valueOf(System.getProperty("cassandra.seed_count_warn_threshold", "20"));
```

Comment:
We can use `parseInt`:
```suggestion
    private static final int seedCountWarnThreshold = Integer.parseInt(System.getProperty("cassandra.seed_count_warn_threshold", "20"));
```